github/Organization.py

```diff
@@ -582,6 +582,42 @@ def create_repo(
             self._requester, headers, data, completed=True
         )
 
+    def create_secret(
+        self,
+        secret_name,
+        unencrypted_value,
+        visibility="all",
+        selected_repository_ids=[],
```

Comment:
It feels more true to our interpretation of the API to call this `selected_repositories` and then allow users to pass either `Repository` objects or ids, and then transform them to a list of IDs when we make the PUT request.

Reply:
@s-t-e-v-e-n-k Thank you for review.
I made changes to pass list of repos, similar approach as in Organization.create_team. Please check it